insights/src/main/kotlin/com/algolia/instantsearch/insights/event/EventUploaderWorkManager.kt

```diff
@@ -38,6 +38,6 @@ internal class EventUploaderWorkManager(
 
             it.setBackoffCriteria(BackoffPolicy.EXPONENTIAL, backOffDelayInSeconds, TimeUnit.SECONDS)
         }.build()
-        WorkManager.getInstance().enqueue(worker)
+        WorkManager.getInstance()!!.enqueue(worker)
```

Comment:
Why ?

Reply:
From the documentation:

```
 /**
     * Retrieves the {@code default} singleton instance of {@link WorkManager}.
     *
     * @return The singleton instance of {@link WorkManager}; this may be {@code null} in unusual
     *         circumstances where you have disabled automatic initialization and have failed to
     *         manually call {@link #initialize(Context, Configuration)}.
     */
```

So I actually want it to crash in case the user disabled automatic initialization, which can then be fixed.

Reply:
So if it's crashing, would the user know why ? Shouldn't we throw an exception or something? The user is only going to see an error when unwrapping a null, without much context

Reply:
True. Although in this case the user would be us, never the client, as we are the one deciding how to initialise the worker manager.

Reply:
Yes, but still, I would have to read the entire code to understand why.

Reply:
Sorry I wasn't clear, I still amended the problem despite my remark. See: c4b7e5b064b71d6d32bf8ea281e75bb7455aacf4

Reply:
Sounds good. Great